planning/behavior_path_planner/autoware_behavior_path_lane_change_module/README.md

```diff
@@ -934,6 +934,16 @@ The following parameters are used to judge lane change completion.
 | `delay_lane_change.min_road_shoulder_width`       | [m]  | double | Width considered as road shoulder if lane doesn't have road shoulder when checking for parked vehicle | 0.5           |
 | `delay_lane_change.th_parked_vehicle_shift_ratio` | [-]  | double | Stopped vehicles beyond this distance ratio from center line will be considered as parked             | 0.6           |
 
+### Terminal Lane Change Path
+
+The following parameters are used to configure terminal lane change path feature.
```

Comment:
Can you add a short explanation of what is the "terminal lane change path feature". I cannot find it explained in the README.